### ISSUE TITLE
Can't build a node  in Xcat

### ISSUE THREAD

**Issue by rathdowney** [xcat2/xcat-core#1943]
Hi,

I'm having issues with building a node on Xcat
I can do rpower the node etc.. but it won't build the image for me
I'm not sure what I am missing in the tables, 
Can someone help me?

**Comment by gurevichmark** [comment]
Can you please provide some additional information:

1. What version of xCAT are you running ? (lsxcatd -a)
2. Did you run `copycds` command to create an osimage ?
3. Can you show the output of the `lsdef` command for the node you are trying to boot ?
4. Can you show the output of the `lsdef -t osimage` command for the image you are trying to boot the node with ?

**Comment by rathdowney** [comment]
Hi Gurevichmark

I am using Version 2.10
I did run copycds to create an osimage

`lsdef` -t node -o d2mipphhpc1176
Object name: d2mipphhpc1176
    arch=x86_64
    bmc=10.125.44.164
    bmcpassword=calvin
    bmcusername=root
    chain=osimage
    currchain=boot
    currstate=boot
    groups=compute,all
    interface=eth0
    ip=10.125.12.164
    mac=D0:43:1E:8D:91:DC
    mgt=ipmi
    netboot=xnba
    nodetype=osi
    os=sles12.1
    postbootscripts=otherpkgs
    postscripts=syslog,remoteshell,syncfiles
    power=ipmi
    profile=compute
    provmethod=sles12.1-x86_64-install-compute
    routenames=vlan2138
    servicenode=d2mixphhpc032
    status=booting
    statustime=10-06-2016 12:03:09
    xcatmaster=d2mixphhpc033`

lsdef -t osimage
sles12.1-x86_64-install-compute  (osimage)
sles12.1-x86_64-netboot-compute  (osimage)

**Comment by gurevichmark** [comment]
Thank you for the information.

1. I noticed that the setting if `xcatmaster` in the output of `lsdef` command has a "`" at the end.
2. Am I correct in assuming that the service node, d2mixphhpc032 is up while compute node d2mipphhpc1176 is being powered on ?
3. xCAT 2.10 had a problem with supporting SLES 12.1 in hierarchical environment. That problem was fixed with pull request #1187, which is part of xCAT 2.12.1 release.
